### DISCUSSION TITLE
After uninstalling zsh...

### DISCUSSION THREAD

**Discussion by darkvoilet** [ohmyzsh/ohmyzsh#10483] [Help] [answered]
Hi. I'm on Ubuntu 20.04, and I watched some tutorial on how to install zsh, but I gave up, so after I installed it, I used the command `sudo apt remove zsh`. Then I rebooted my computer and Ubuntu welcomed me with the window you get when making a new user.
I kept on with the instructions and when I finished I figured I created a new user. When I went to `Settings>Users` I saw that only one user was there - the one I just created. I got very worried I had lost my old files, but I went to the home directory, I had to users there, and I still had all my old files.
I rebooted again and in the login screen I only had one user - the one I created before. I thought I won't be able to login to my old user with my files, but then I clicked `not listed?` and typed my old user's name and password, and got back.
The thing is that my old user doesn't have administrative permissions and I can't login easily to my user. Then someone asked me: "Why did I uninstall my shell..."

Someone please help me with this, I really need this problem to be fixed.
Thanks!
![Screenshot from 2021-12-06 12-46-17](https://user-images.githubusercontent.com/95619567/144833527-cf9d3f3a-711a-4eb1-a574-ac97f74fa4f3.png)

**Comment by mcornella** [comment, marked as answer]
You need to reinstall zsh, so you can login with your user, or change the shell of the user to `bash` or whatever it was before. This command will work, provided you can login as root or have `sudo` enabled for your new user:
```sh
sudo chsh -s $(which bash) <user>
```
You need to substitute `<user>` for your username.
